spec/lib/api/2.0/notification-spec.js

```diff
@@ -88,187 +88,160 @@ describe('Http.Api.Notification', function () {
     });
 
     describe('POST /notification/progress', function () {
-        beforeEach(function() {
-            sinon.stub(notificationApiService, 'postNotification').resolves();
-        });
+        describe('should return 200', function() {
```

Comment:
It doesn't make sense to create such kind of test suites. Any reason to do this?
And if you are surely want to create a test suite here, the test suite name need be changed to a better one. The text `should xxx` is usually used only for test case.